### PR TITLE
Task/3697 vm details page design

### PR DESCRIPTION
**Pull request by tiwariav**

### CODE REVIEW

**Comment by verysanghee** [comment]
design very well realised, thanks for the great job! @tiwariav Only need to recheck for the btns and links to work

**Comment by verysanghee** [comment]
I think since we don't see the Terminate btn becoming "active" for significant length, we should change the btn hover into red warning color #eb4d5c

**Comment by verysanghee** [comment]
for "copied" tooltip, can we change the background color into same as other icons? I find #000 is too much prominent.

**Comment by verysanghee** [comment]
translation needs to be updated before merging.

File: hosting/locale/de/LC_MESSAGES/django.po [hunk 1]
```diff
@@ -399,38 +399,90 @@ msgstr "Möchtest Du den Schlüssel löschen?"
...
-msgstr "Abrechnungen"
+#, fuzzy
+#| msgid "Settings"
+msgid "VM Settings"
+msgstr "Einstellungen"
 
-msgid "Ip not assigned yet"
-msgstr "Ip nicht zugewiesen"
+msgid "Copied"
+msgstr ""
```
Comment (verysanghee) [inline hosting/locale/de/LC_MESSAGES/django.po]: "Kopiert"

File: hosting/locale/de/LC_MESSAGES/django.po [hunk 2]
```diff
@@ -399,38 +399,90 @@ msgstr "Möchtest Du den Schlüssel löschen?"
...
+msgid "See Invoice"
+msgstr "Rechnung"
+
+#, fuzzy
+#| msgid "Your SSH Keys"
+msgid "Your VM is"
+msgstr "Deine SSH Keys"
+
+msgid "Pending"
+msgstr ""
```
Comment (verysanghee) [inline hosting/locale/de/LC_MESSAGES/django.po]: "In Vorbereitung"

File: hosting/locale/de/LC_MESSAGES/django.po [hunk 3]
```diff
@@ -399,38 +399,90 @@ msgstr "Möchtest Du den Schlüssel löschen?"
...
+
+msgid "Online"
+msgstr ""
+
+msgid "Failed"
+msgstr ""
```
Comment (verysanghee) [inline hosting/locale/de/LC_MESSAGES/django.po]: "Fehlgeschlagen"

File: hosting/locale/de/LC_MESSAGES/django.po [hunk 4]
```diff
@@ -399,38 +399,90 @@ msgstr "Möchtest Du den Schlüssel löschen?"
...
+
+msgid "Something doesn't work?"
+msgstr ""
+
+msgid "We are here to help you!"
+msgstr ""
+
+msgid "CONTACT"
+msgstr ""
```
Comment (verysanghee) [inline hosting/locale/de/LC_MESSAGES/django.po]: "Kontakt"

File: hosting/locale/de/LC_MESSAGES/django.po [hunk 5]
```diff
@@ -399,38 +399,90 @@ msgstr "Möchtest Du den Schlüssel löschen?"
...
+msgid "Failed"
+msgstr ""
+
+#, fuzzy
+#| msgid "Create VM"
+msgid "Terminate VM"
+msgstr "Neue VM"
+
+msgid "Support / Contact"
+msgstr ""
```
Comment (verysanghee) [inline hosting/locale/de/LC_MESSAGES/django.po]: "Support / Kontakt"

File: hosting/locale/de/LC_MESSAGES/django.po [hunk 6]
```diff
@@ -399,38 +399,90 @@ msgstr "Möchtest Du den Schlüssel löschen?"
...
+
+msgid "Something doesn't work?"
+msgstr ""
```
Comment (verysanghee) [inline hosting/locale/de/LC_MESSAGES/django.po]: "Etwas funktioniert nicht?"

File: hosting/locale/de/LC_MESSAGES/django.po [hunk 7]
```diff
@@ -399,38 +399,90 @@ msgstr "Möchtest Du den Schlüssel löschen?"
...
+
+msgid "We are here to help you!"
+msgstr ""
```
Comment (verysanghee) [inline hosting/locale/de/LC_MESSAGES/django.po]: "Wir sind hier, um Dir zu helfen!"

File: hosting/locale/de/LC_MESSAGES/django.po [hunk 8]
```diff
@@ -399,38 +399,90 @@ msgstr "Möchtest Du den Schlüssel löschen?"
...
+
+msgid "CONTACT"
+msgstr ""
+
+msgid "BACK TO LIST"
+msgstr ""
```
Comment (verysanghee) [inline hosting/locale/de/LC_MESSAGES/django.po]: "Zurück zur Liste"

File: hosting/urls.py [hunk 9]
```diff
@@ -10,6 +11,7 @@
     SSHKeyDeleteView, SSHKeyCreateView, SSHKeyListView, SSHKeyChoiceView
 
 urlpatterns = [
+    # url(r'test/?$', TemplateView.as_view(template_name='hosting/virtual_machine_detail.html')),
```
Comment (pcoder) [inline hosting/urls.py]: Please remove the above commented line.

**Pull request merged**